src/runtime/components/elements/Alert.vue

```diff
@@ -1,8 +1,12 @@
 <template>
   <div :class="alertClass" v-bind="attrs">
     <div class="flex" :class="[ui.gap, { 'items-start': (description || $slots.description), 'items-center': !description && !$slots.description }]">
-      <UIcon v-if="icon" :name="icon" :class="ui.icon.base" />
-      <UAvatar v-if="avatar" v-bind="{ size: ui.avatar.size, ...avatar }" :class="ui.avatar.base" />
+      <slot name="icon" :icon="icon" :ui="ui.icon">
```

Comment:
I'm following up on this `ui` added to the slot, I don't think its a good idea to merge this in the current version as it would require to do the same on all the other slots.

I'm currently working on the `v3` and have taken this into account to do it everywhere as you can read in #1289.

Can we remove it so we can merge this PR?